lib/devise_security_extension/hooks/session_limitable.rb

```diff
@@ -18,7 +18,7 @@
   env   = warden.request.env
 
   if record.respond_to?(:unique_session_id) && warden.authenticated?(scope) && options[:store] != false
-    if record.unique_session_id != warden.session(scope)['unique_session_id'] && !env['devise.skip_session_limitable']
+    if record.unique_session_id != warden.session(scope)['unique_session_id'] && !env['devise.skip_session_limitable'] && !warden.request.session.key?('impersonator_id')
```

Comment:
Why is `impersonator_id` meaningful? This is another good reason to create methods that describe what they do instead of working with hash keys, array elements, etc.